<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1538</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>900</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_next">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>550</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_class_ruler">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>279</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Ruler</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_class_border">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>340</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Black Border</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_comments">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>480</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Comments (Optional)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_class_stain">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Stain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_prev">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>550</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_img_name">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>210</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>620</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 170, 127);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Save Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_img_save">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>680</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 0)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_class_subtle_ruler">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Subtle Ruler</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_class_subtle_border">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>340</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Subtle Border</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_class_subtle_stain">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>390</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Subtle Stain</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
